src/EditorFeatures/TestUtilities2/Intellisense/ModernCompletionTestState.vb

```diff
@@ -183,21 +183,30 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             Task.WaitAny(task1, task2)
         End Sub
 
-        Public Overrides Async Function AssertCompletionSession(Optional projectionsView As ITextView = Nothing) As Task
+        Public Overrides Function AssertCompletionSession(Optional projectionsView As ITextView = Nothing) As Task
             Dim view = If(projectionsView, TextView)
 
-            Await WaitForAsynchronousOperationsAsync()
```

Comment:
💡 This one would also benefit from `AssertNoAsynchronousOperationsRunning`

Reply:
Why? The async completion does not involve features with those attributes: 
FeatureAttribute.EventHookup, FeatureAttribute.CompletionSet, FeatureAttribute.SignatureHelp 
Why is the benefit of waiting for them? And then for asserting that nothing is running?

---
In reply to: [314785676](https://github.com/dotnet/roslyn/pull/38021#discussion_r314785676) [](ancestors = 314785676)